pyproject.toml

```diff
@@ -99,6 +99,7 @@ disallow_untyped_defs = false
 addopts = [
   # `pytest-xdist`:
   "--numprocesses=auto",
+  "--dist=worksteal",
```

Comment:
Such settings would benefit from having an inline code comment with the motivation.

Reply:
Thanks for the suggestion! Addressed in https://github.com/jazzband/pip-tools/pull/1973